nmdc_schema/migrators/migrator_from_X_to_PR10.py

```diff
@@ -63,7 +63,8 @@ def upgrade(self):
 
     def add_type_to_inlined_classes(self, document: dict, slot: str, uri: str):
         r"""
-        Adds a type slot to each inlined instance of an NMDC class in the biosmpale_set, study_set, and extraction_set collections. 
+        Adds a type slot to each inlined instance of an NMDC class in the biosmpale_set, study_set, extraction_set collections, 
+        mags_set, metabolomics_analysis_set, and metaproteomics_analysis_set. 
```

Comment:
Now that the list is longer, I think you meant for the "collections" word to be at the very end of the comment.

Reply:
@eecavanna I just removed the word "collections" entirely. I think this makes sense too.